core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -79,6 +80,8 @@ trait LoadBalancer {
   def publish(action: ExecutableWhiskActionMetaData, msg: ActivationMessage)(
     implicit transid: TransactionId): Future[Future[Either[ActivationId, WhiskActivation]]]
 
+  def allInvokers: Future[IndexedSeq[(InstanceId, InvokerState)]]
```

Comment:
Why should `allInvokers` be part of this interface? It should only be referenced internally (except that it currently is referenced as a health check endpoint on controller `/invokers`, which should delegate to the loadbalancer IMHO.